### ISSUE TITLE
mentioned promise module didn't work for me

### ISSUE THREAD

**Issue by leonardodino** [threepointone/react-act-examples#2]
I replaced:

```javascript
import PromisePonyfill from 'promise'
PromisePonyfill.enableSynchronous()
global.Promise = PromisePonyfill
```

with:

```javascript
import { ZalgoPromise } from 'zalgo-promise'
global.Promise = ZalgoPromise
```

in my CRA `setupTests.js` and things started working.
just not for the `await` syntax, yet. but `.then` works.

**Comment by threepointone** [comment]
I've update the doc and the examples with the new async api, available in v16.9.0-alpha.0. Have a look and lemme know?

**Comment by leonardodino** [comment]
😍thanks so much for your work on this Sunil!

I haven't had the time yet to test, I've moved to another project.

But I saw some of the announcements!
I think you can close this issue. (:
